pom.xml

```diff
@@ -42,13 +42,6 @@
             <scope>test</scope>
         </dependency>
 
-        <dependency>
-            <groupId>org.spockframework</groupId>
-            <artifactId>spock-spring</artifactId>
-            <version>1.2-groovy-2.4</version>
-            <scope>test</scope>
-        </dependency>
-
```

Comment:
did you remove Spock?

Reply:
NO, i have removed the latest one, but above the removed code you can see its still has the spock plugin.